ports/lhasa/vcpkg.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "lhasa",
+  "version-semver": "0.4.0",
```

```suggestion
  "version": "0.4.0",
```
If there is no special requirement, we do not recommend using the semantics of `semver`. 
If you must use `semver`, please ignore my suggestion of `${VERSION}` above, it is not suitable for unexpected `version`.